v3/src/components/graph/hooks/use-plot.ts

```diff
@@ -145,7 +150,7 @@ export const usePlotResponders = (props: IPlotResponderProps) => {
         })
         callRefreshPointPositions(false)
       }
-    })
+    }) || (() => true)
```

Comment:
```suggestion
    })
```
Rather than this, you should be able to just add a `?.` to line 154 below:
```typescript
    return () => disposer?.()
```

Reply:
Nope. TS1109: Expression expected.
Leave as is.